content/blog/2019/05/2019-05-30-becoming-contributor-newbie-tickets.adoc

```diff
@@ -0,0 +1,243 @@
+
+The https://wiki.jenkins.io/display/JENKINS/Beginners+Guide+to+Contributing[beginners guide to contributing to Jenkins]
+is probably the best starting point, and a reference link to keep handy. It points to different repositories, tools
+(such as the issue tracker) and to governance documents. It also describes best practices for commit messages, code style
+conventions, PR guidelines, etc.
+
+Once we get a general understanding of the former and want to actually start coding, we may get stuck trying to come up
+with something to work on.
+
+Visiting the https://issues.jenkins-ci.org/projects/JENKINS/issues[Jenkins issue tracker], feels like the natural next
+step, since it is full of potential bugs and enhancements that have already been reported by the community. However, it
+is quite easy to feel overwhelmed by the possibilities listed there. Bear in mind that in a 10+-year-old project like
+this, most of the things that are reported are tricky for a newcomer to work on. For that reason, filtering by
+https://issues.jenkins-ci.org/browse/WEBSITE-625?jql=labels%20%3D%20newbie-friendly[newbie-friendly tickets] is probably
+the best idea.
+
+.Screenshot displaying the list of newbie-friendly tickets in the Jenkins Jira
+image::/images/post-images/2019-05-30-becoming-contributor/list-newbie-tickets.png[]
+
+=== Selecting a ticket
+
+In my case, I spent some time reviewing the `newbie-friendly` tickets, until https://issues.jenkins-ci.org/browse/JENKINS-56477[I found one]
+that seemed interesting to me and also looked like something I would be able to fix:
+
+.Screenshot of the ticket I decided to work on
+image::/images/post-images/2019-05-30-becoming-contributor/jenkins-newbie-jira-ticket-selected.png[]
+
+=== Processes
+
+At this stage, when we have decided to take ownership of a ticket, it’s a good practice to let the rest of the community
+know that we are planning to start working on it. We can do so easily, by assigning the ticket to ourselves (see the
+“_Assign_” button below the ticket summary).
+
+Assigning the ticket to ourselves in the Jenkins Jira will allow any other contributors to know that we are planning to
+take care of the ticket; and in case they are also interested in contributing to it, they will know who to reach if they
+want to coordinate work or ask for status. That said, it is worth mentioning that assigning a ticket to yourself does
+not mean that other contributors cannot work on it from then onwards. Jenkins is an open-source project and anyone is
+welcome to create their own PRs, so anybody can propose their own solution to the ticket. But as you can guess, if the
+ticket is assigned to somebody, most people will probably reach the assignee before starting to work on it.
+
+Related to the former, it is also important to bear in mind that we should not postpone work on the ticket for too long
+once we have assigned the ticket to ourselves. Other potential contributors might be discarding the ticket because they
```

Comment:
I think you mean "ignoring" or possibly "disregarding" rather than "discarding".

```suggestion
once we have assigned the ticket to ourselves. Other potential contributors might be ignoring the ticket because they
```